src/features/import/utils/createPreviewData.ts

```diff
@@ -54,6 +54,17 @@ export default function createPreviewData(
         });
       }
 
+      if (column.kind === ColumnKind.ENUM) {
+        column.mapping.forEach((mappedColumn) => {
+          if (mappedColumn.value === row[colIdx]) {
+            personPreviewOp.data = {
+              ...personPreviewOp.data,
+              [`${column.field}`]: row[colIdx],
+            };
+          }
+        });
+      }
```

Comment:
Please add a test case to `createPreviewData.spec.ts` that encompasses this. Maybe this code is related to the fact that the preview doesn't seem to be working (see separate comment) and you can try to reproduce that in a test and TDD a solution?

Reply:
The test should verify that mapping empty values work as expected. Right now it does not seem to be working. The store looks ok after mapping an empty value, but the preview does not render anything. I'm not sure if it's here, or if it's the UI.

![image](https://github.com/user-attachments/assets/dc798156-6dc6-4446-a9e0-a0669bdcabf7)

Reply:
Moved mapping to createPreviewData (previously in EnumPreview component) and added test for this.

Reply:
I have yet to review the code, but I just tried this in the build and can confirm that it's working. Nice work!